include/net/nrf_cloud_cell_pos.h

```diff
@@ -12,6 +12,7 @@
  */
 
 #include <zephyr.h>
+#include <modem/lte_lc.h>
```

Comment:
@jayteemo why we need to include LTE_LC header?
CELL_POS should be runnable WITHOUT using LTE_LC lib.

Reply:
it is just the header, which is needed for ` struct lte_lc_cells_info` (the struct that contains cell measurement data for MCELL).
see change to `nrf_cloud_cell_pos_request`